password_security/models/res_users.py

```diff
@@ -32,16 +32,37 @@ def write(self, vals):
             vals["password_write_date"] = fields.Datetime.now()
         return super().write(vals)
 
+    @api.model
+    def _get_all_password_params(self):
+        params = self.env["ir.config_parameter"].sudo()
+        res = {
+            "minlength": int(
+                params.get_param("auth_password_policy.minlength", default=0)
+            ),
+            "expiration_days": int(
+                params.get_param("password_security.expiration_days", default=60)
+            ),
+            "minimum_hours": int(
+                params.get_param("password_security.minimum_hours", default=60)
+            ),
+            "history": int(params.get_param("password_security.history", default=30)),
+            "lower": int(params.get_param("password_security.lower", default=1)),
+            "upper": int(params.get_param("password_security.upper", default=1)),
+            "numeric": int(params.get_param("password_security.numeric", default=1)),
+            "special": int(params.get_param("password_security.special", default=1)),
+        }
```

Comment:
These defaults don't match previous versions. Change all default back to zero and get rid of this method entirely?